parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -2,10 +2,11 @@
 Description=Kubelet
 ConditionPathExists=/usr/local/bin/kubelet
 Wants=network-online.target
-After=network-online.target
+After=network-online.target containerd.service
```

Comment:
can we sanity check this in standalone? 

I'd also argue to add containerd to Wants to avoid races.

Reply:
possibly even Requires vs Wants...up to you